Test Utilities/Doubles/Action.swift

```diff
@@ -0,0 +1,9 @@
+@testable import PusherChatkit
+
+public class DummyAction: Action {
```

Comment:
I'd probably name this `FakeAction` (its not inheriting from `DummyBase` and doesn't cause any issues if interacted with).

Reply:
I will rename this.